boot/core/src/boot/task/built_in.clj

```diff
@@ -103,7 +103,6 @@
             (core/empty-dir! tmp)
             (util/info "Checking out %s...\n" path)
             (pod/unpack-jar (.getPath file) tmp
-              :cache false
```

Comment:
Ah, I see. So the bizarre caching behavior of the `URLConnection` isn't an issue anymore because we're not using `URLConnection`s anymore, right?

:metal: